patients.html

```diff
@@ -54,7 +54,7 @@
       <div class="row justify-content-center">
         <div class="col-sm-3 col-md-3 p-sm-2 p-md-5 text-center">
           <a href="#">
-          <amp-img src="/assets/uploads/patients-tooth.png"
+          <amp-img src="/assets/images/patients-tooth.webp"
```

Comment:
all images are now in the upload directory, and we don't need to use webp images, the build automatically creates those for us

Reply:
@konarie1990